scalding-core/src/main/scala/com/twitter/scalding/macros/impl/CaseClassBasedSetterImpl.scala

```diff
@@ -15,20 +15,16 @@
  */
 package com.twitter.scalding.macros.impl
 
-import scala.reflect.macros.Context
+import scala.reflect.macros.{ Context, whitebox }
```

Comment:
It's whitebox macros because this is how the original Context being implemented.

```
package object macros {
  /** The Scala macros context.
   *
   *  In Scala 2.11, macros that were once the one are split into blackbox and whitebox macros,
   *  with the former being better supported and the latter being more powerful. You can read about
   *  the details of the split and the associated trade-offs in the [[http://docs.scala-lang.org/overviews/macros.html Macros Guide]].
   *
   *  `scala.reflect.macros.Context` follows this tendency and turns into `scala.reflect.macros.blackbox.Context`
   *  and `scala.reflect.macros.whitebox.Context`. The original `Context` is left in place for compatibility reasons,
   *  but it is now deprecated, nudging the users to choose between blackbox and whitebox macros.
   */
  @deprecated("Use blackbox.Context or whitebox.Context instead", "2.11.0")
  type Context = whitebox.Context
}
```